src/test/java/org/elasticsearch/example/realm/CustomRealmIT.java

```diff
@@ -44,6 +44,8 @@
 import java.util.Collections;
 import java.util.List;
 
+import static com.carrotsearch.randomizedtesting.RandomizedTest.randomAsciiOfLength;
+import static com.carrotsearch.randomizedtesting.RandomizedTest.randomAsciiOfLengthBetween;
```

Comment:
These aren't right, these should be replaced by `randomAlphaOfLength` from `ESTestCase`. The reason for this change is because `randomAsciiOfLength` is a lie, it is not random ASCII, it's only random lowercase and uppercase alphabetic characters.